app/i18n/locales/ko_KR.json

```diff
@@ -16,6 +16,8 @@
   "loading.screen.connectingToNetworkMessage": "!!!Connecting to network",
   "loading.screen.reconnectingToNetworkMessage": "!!!Network connection lost - reconnecting",
   "loading.screen.syncingBlocksMessage": "!!!Syncing blocks",
+  "profile.languageSelect.form.languageSelectLabel": "!!!Select your language",
+  "profile.languageSelect.form.submitLabel": "!!!Continue",
```

Comment:
@darko-mijic thanks for pointing this out! All manually added translation keys are now removed.